testplan/exporters/testing/http/__init__.py

```diff
@@ -25,7 +25,8 @@ class HTTPExporterConfig(ExporterConfig):
     def get_options(cls):
         return {
             ConfigOption('url'):
-                Regex(r'^http[s]?://[\w\d_-]+(:\d{2,5})?.+$', flags=re.I)
+                Regex(r'^http(s)?://(\w+:\w+@)?[\w\.-]+(:\d{2,5})?(/[\w-]*)*$',
```

Comment:
yes it's aim to avoid obvious wrong format. let me have a look at it again and and verify that if we need a validator if the regex is too long, also let me try if we need http prefix.

Reply:
OK great thanks. I don't think you should need to spend a lot of time on this. The validation is useful but the most important thing is not to reject possibly valid URLs, it's acceptable if some bad URLs get through because like you say it only means they will fail later.

Reply:
seems it's not easy to write a url regex simply in one line to avoid obvious wrong format. for example:
http://192.168.1
http://www.-abc.com
http://www.a--bc.com
And I also tested that generally the requests library needs a 'http://' prefix to work with web server. So, at last, I think it's better use validators, it can detect all the wrong format.